components/common/navbar/Navbar.tsx

```diff
@@ -1,42 +1,168 @@
-        <nav className="md:ml-auto flex flex-wrap items-center text-base justify-center">
-          <Link href="/">
-            <a className="mr-5 hover:text-gray-900"> Home</a>
-          </Link>
-          <Link href="/">
-            <a className="mr-5 hover:text-gray-900">Community</a>
-          </Link>
-          <Link href="/events">
-            <a className="mr-5 hover:text-gray-900">Event</a>
-          </Link>
-          <Link href="/notices">
-            <a className="mr-5 hover:text-gray-900">Notice</a>
-          </Link>
-          <Link href="/">
-            <a className="mr-5 hover:text-gray-900">About</a>
-          </Link>
-        </nav>
+    <header className="sticky top-0 flex flex-row w-full h-14 justify-between items-center shadow-noOffset z-10">
+      <div className="w-28 h-12 flex flex-row items-center ml-9">
+        <Image src={fcscLogo} alt="FCSC Logo" className="w-max" />
+      </div>
+      <div className="flex flex-row justify-end  items-center w-4/6 h-14 bg-white">
+        <div className="flex flex-row mr-2">
+          {navs.map((nav, key) => (
+            <div
+              key={key}
+              className={`flex flex-row hidden lg:block  items-center justify-center mr-10 h-7 rounded-md text-center font-semibold 
+              ${
+                router.pathname == nav.href ? 'bg-fcsc-orange w-16' : 'bg-white'
+              } ${nav.text == 'Community' ? 'w-28' : ' w-16'}
+              `}
+            >
+              <Link href={nav.href}>
+                <a
+                  className={`${
+                    router.pathname == nav.href
+                      ? 'text-white w-28'
+                      : 'text-fcsc-blue'
+                  }`}
+                >
+                  {nav.text}
+                </a>
+              </Link>
+            </div>
+          ))}
+        </div>
+        <div
+          className="block lg:hidden z-20 cursor-pointer transition ease-in  sm:mr-3  md:mr-3 lg:mr-0"
+          onClick={toggleNav}
+        >
+          {!isOpen ? (
+            <HiMenuAlt2 className="h-6 w-6 hover:text-gray-default transition ease-in" />
+          ) : (
+            ''
+          )}
+        </div>
+        <AnimatePresence exitBeforeEnter>
+          {isOpen ? (
+            <motion.div
+              initial="initial"
+              animate="animate"
+              exit="exit"
+              variants={mobileNavVariants}
+              className={`fixed bg-white top-0 left-0 bottom-0 z-10 min-h-screen ${
+                isOpen ? `w-full` : `w-0`
+              } bg-blue flex flex-col items-center justify-center`}
+            >
+              <div
+                className="absolute top-4 right-4 cursor-pointer"
+                onClick={toggleNav}
+              >
+                {isOpen ? (
+                  <RiCloseFill className="h-6 w-6 hover:text-gray-light text-fcsc-blue transition ease-in" />
+                ) : (
+                  ''
+                )}
```

Comment:
use `&` like this `isOpen && <RiCloseFill className="h-6 w-6 hover:text-gray-light text-fcsc-blue transition ease-in" />`